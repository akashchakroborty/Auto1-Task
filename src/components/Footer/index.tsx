import React from 'react';
import { makeStyles, createStyles } from '@mui/styles';
import { Theme, Typography } from '@mui/material';

const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    footer: {
      borderTop: `1px solid ${theme.palette.divider}`,
      marginTop: 64,
      padding: 16,
      height: 80,
      display: 'flex',
      justifyContent: 'center',
      alignItems: 'center',
    },
  }),);

const Footer: React.FC = () => {
  const classes = useStyles();
  return (
    <footer className={classes.footer}>
      <Typography variant="body2" color="textSecondary" align="center">
        &copy; AUTO1 Group 2022
      </Typography>
    </footer>
  );
};

export default Footer;
