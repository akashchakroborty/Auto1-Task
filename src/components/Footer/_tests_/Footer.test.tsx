/* eslint-disable */
import React from 'react';
import { shallow } from 'enzyme';
import Footer from '..';

jest.mock('@mui/styles', () => {
  const Styles = jest.requireActual('@mui/styles');
  const theme = { palette: { divider: {} } };

  return {
    ...Styles,
    makeStyles: (func: any) => {
      return Styles.makeStyles(func.bind(null, theme));
    },
  };
});

describe('should test the Footer Component', () => {
  test('should render the component', () => {
    const component = shallow(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
