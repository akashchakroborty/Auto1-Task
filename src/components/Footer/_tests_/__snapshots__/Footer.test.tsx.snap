// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test the Footer Component should render the component 1`] = `
<footer
  className="makeStyles-footer-1"
>
  <ForwardRef(Typography)
    align="center"
    color="textSecondary"
    variant="body2"
  >
    Â© AUTO1 Group 2022
  </ForwardRef(Typography)>
</footer>
`;
